topics/duckduckgo/index.md

```diff
@@ -0,0 +1,9 @@
+---
+topic: duckduckgo
+display_name: DuckDuckGo
+logo: duckduckgo.png
+short_description: DuckDuckGo is an Internet search engine that emphasizes protecting searchers' privacy and avoiding the filter bubble of personalized search results.
```

Comment:
We try to keep the short and long descriptions different –– it looks like the CI failed on two things:

```

  2) Failure:
topics::duckduckgo topic#test_0015_has a short_description that differs from the body [/home/travis/build/github/explore/test/topics_test.rb:173]:
body and short description should differ.
Expected "DuckDuckGo is an Internet search engine that emphasizes protecting searchers' privacy and avoiding the filter bubble of personalized search results." to not be equal to "DuckDuckGo is an Internet search engine that emphasizes protecting searchers' privacy and avoiding the filter bubble of personalized search results.".
  3) Failure:
topics::duckduckgo topic#test_0023_has a valid short_description [/home/travis/build/github/explore/test/topics_test.rb:279]:
must have a short_description no more than 130 characters (currently 148)
```

How would you feel about changing the `short_description` to the following:

```diff
+ short_description: DuckDuckGo is an Internet search engine that prioritizes privacy by not tracking users.
- short_description: DuckDuckGo is an Internet search engine that emphasizes protecting searchers' privacy and avoiding the filter bubble of personalized search results.
```

Reply:
took yours.

Reply:
Took your `short_description` suggestion. https://github.com/theel0ja/explore/commit/d1652f5c51fbc0a50c6bb52be10fad1b9ca3cefe